lib/util/http.ts

```diff
@@ -0,0 +1,26 @@
+import got from './got';
+
+export class Http {
+  hostType: string;
+
+  options: any;
```

Comment:
We should add type, otherwise it would be difficult to find all changes required to mograte to got v10.
```suggestion
  readonly options: HttpOptions;
```